### PR TITLE
Add jump link

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

File: assets/sass/patterns/molecules/jump-menu.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  &:hover {
+    color: $color-text;
+  }
+}
+
+.jump-menu__active {
+  @include margin(-2, "left");
+  color: $color-text;
+  border-left: 2px solid $color-primary;
```
Comment (discodavey) [inline assets/sass/patterns/molecules/jump-menu.scss:32]: Needs to be in alpha order

**Review by discodavey — COMMENTED** [review]
There is some spacing still that needs removing on article-research mustache file

**Comment by discodavey** [comment]
@QueenKraken This is part of some bigger work with putting tabs into the journal like we have in EPP. Is this still a priority to do after the decision was made last week to scale back Journal work?

**Comment by discodavey** [comment]
I know that we are using the view selector code for the jump links but is everyone happy with this approach or would we consider using the existing code for both i.e. could this be done or is there a better way of doing this?

This is purely a throwing this up in the air. There is no right or wrong answer to this

File: assets/sass/patterns/molecules/jump-menu.scss (hunk 1 above)
Comment (viktoria-mahmud) [reply assets/sass/patterns/molecules/jump-menu.scss:31]: Done.

**Comment by viktoria-mahmud** [comment]
Done.

> There is some spacing still that needs removing on article-research mustache file

**Comment by viktoria-mahmud** [comment]
We need to support View Selector but it will be removed later, so it will be easier to remove the logic. 

> I know that we are using the view selector code for the jump links but is everyone happy with this approach or would we consider using the existing code for both i.e. could this be done or is there a better way of doing this?
> 
> This is purely a throwing this up in the air. There is no right or wrong answer to this

**Review by discodavey — APPROVED** [review]

**Pull request merged**